src/open_inwoner/pdc/migrations/0049_auto_20230222_0901.py

```diff
@@ -8,7 +8,7 @@ def truncate_product_summary(apps, schema_editor):
     for product in MyModel.objects.all():
         if len(product.summary) > 300:
             product.summary = product.summary[:300]
-            product.save()
+            product.save(update_fields=["summary"])
```

Comment:
I did a lot of changes concerning migrations..thanks for that!

Reply:
I'd probably been fine with the optimisations but its a good pattern to not do unnecessary work, especially in bulk operations.